ckanext/qgov/common/resources/resource_types.json

```diff
@@ -15,6 +15,7 @@
             "application/xml"
         ]
     },
+    "archive_types": ["application/zip", "application/x-tar"],
```

Comment:
gzip?

Reply:
It's a non-issue really, since we don't allow `.gz` extensions anyway. Nor `.tgz`; I included the TAR format basically because I could and because it seemed like a better test.